src/Dropdown.php

```diff
@@ -112,7 +113,15 @@ protected function renderItems($items, $options = [])
             $itemOptions = ArrayHelper::getValue($item, 'options', []);
             $linkOptions = ArrayHelper::getValue($item, 'linkOptions', []);
 //            $linkOptions['tabindex'] = '-1';
```

Comment:
That is not about this pull request but... any idea why this commented out line is there?

Reply:
I was wondering the same

Reply:
Changed in https://github.com/yiisoft/yii2-bootstrap4/commit/61ddd1bcbd7381d734b756b48ca26a147e9bf1ec

Reply:
I think we may safely delete it.

Reply:
`$itemOptions` is also unsued, or am I overlooking anything?